server/tests-py/queries/remote_schemas/remote_relationships/setup_remote_rel_joining_singleton_with_array.yaml

```diff
@@ -0,0 +1,25 @@
+type: bulk
+args:
+- type: create_remote_relationship
+  args:
+    name: remoteUsers
+    table: profiles
+    hasura_fields:
+      - id
+    remote_schema: my-remote-schema
+    remote_field:
+      users:
+        arguments:
+          user_ids: "$id"
+
+- type: create_remote_relationship
+  args:
+    name: remoteUsersMultipleIds
+    table: profiles
+    hasura_fields:
+      - id
+    remote_schema: my-remote-schema
+    remote_field:
+      users:
+        arguments:
+          user_ids: ["$id"]
```

Comment:
Was support for providing a mapping like this included the current PR? Or did it exist already?

Reply:
The support for the `user_ids: ["$id"]` already existed before this PR. I added a test case to make sure that the older behaviour remains the same.